api.planx.uk/server.ts

```diff
@@ -38,19 +38,18 @@ useSwaggerDocs(app);
 
 app.set("trust proxy", 1);
 
-app.use((req, res, next) => {
-  res.header("Access-Control-Allow-Origin", req.headers.origin);
-  res.header(
-    "Access-Control-Allow-Headers",
-    "Origin, X-Requested-With, Content-Type, Accept",
-  );
-  next();
-});
-
 app.use(
   cors({
     credentials: true,
     methods: "*",
+    origin: process.env.EDITOR_URL_EXT,
```

Comment:
I suspect that we'll need to change this to be "pizzas + staging" if we're on the staging API, either here or specifically in the auth middleware somewhere. I'd like to merge to staging to test this before implementing anything just now just to double check.